data-serving/data-service/schemas/cases.schema.json

```diff
@@ -1,36 +1,36 @@
 {
   "$jsonSchema": {
+    "bsonType": "object",
+    "additionalProperties": false,
     "required": [
       "_id",
       "location",
       "source",
-      "metadata"
+      "revisionMetadata"
     ],
     "properties": {
-      "id": {
+      "_id": {
         "bsonType": "objectId"
       },
       "demographics": {
         "bsonType": "object",
+        "additionalProperties": false,
         "properties": {
           "age": {
             "bsonType": "object",
+            "additionalProperties": false,
             "properties": {
-              "years": {
-                "bsonType": "double",
-                "minimum": -0.75,
-                "maximum": 200.0
-              },
               "range": {
                 "bsonType": "object",
+                "additionalProperties": false,
                 "properties": {
                   "start": {
-                    "bsonType": "int",
+                    "bsonType": "double",
```

Comment:
Can you tl;dr why this is a double? Something meta, or do we have ages like 34.5/(2 years 3 months)?